src/apis/gpib.js

```diff
@@ -1,27 +1,27 @@
-import axios from "axios";
+import axios from 'axios';
```

Comment:
Don't we use double quote?  Can we add the same prettier?

Reply:
Yep done